### ISSUE TITLE
Console output does not support emoji. Some mods use them in messages.

### ISSUE THREAD

**Issue by RDKRACZ** [MultiMC/MultiMC5#3981]
### Operating System

Linux

### Description of bug

The log doesn't display emoji properly. Example message:
[Render thread/INFO]: Krypton is now accelerating your Minecraft client's networking stack 🚀

And how it is displayed:
![Screenshot from 2021-07-31 16-33-11](https://user-images.githubusercontent.com/43277609/127743153-c19c5452-2e74-4b42-872c-29945ae24b54.png)
Notice the rectangle at the end.

### Steps to reproduce

Add Krypton to your mods and observe the log.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by RDKRACZ** [comment]
> font you are using not supporting emoji

This can't be true since the issue only occurs with MultiMC - I toggled log preview in the official launcher to check this.
I'm using the default fonts for both machines where this problem exists:
-Arch:
![Screenshot from 2021-08-28 16-37-04](https://user-images.githubusercontent.com/43277609/131221545-71cc37fb-d7ff-46e7-b2a6-8ba11b6d712b.png)
-Ubuntu:
![image](https://user-images.githubusercontent.com/43277609/131221518-fef3d659-ee8f-49e0-a19b-4b8887551742.png)

**Comment by peterix** [comment]
No. Vanilla launcher uses some bundled font you don't even have. And it's entirely irrelevant. Install and use a font that works.

**Comment by peterix** [comment]
This has absolutely nothing to do with MultiMC and everything to do with your system being broken. 
We do not ship fonts.

**Comment by RDKRACZ** [comment]
>This has absolutely nothing to do with MultiMC and everything to do with your system being broken

Are you telling me that both Pop and Endeavor OS use fonts that don't support emojis?

>Install and use a font that works

Which one do You use?

**Comment by peterix** [comment]
I don't remember. But you can change the console font in the settings.

**Comment by RDKRACZ** [comment]
Ok so I changed console font in MultiMC settings a few times and none of them seem to display emojis properly. Then I tested the same fonts (ex. Source Code Pro, Hack, FiraCode) in a terminal, and the 🚀, as well as other emojis, is visible.

**Comment by kb-1000** [comment]
Works for me. If it matters, I use JetBrains Mono...
Well, I guess it depends on the font

**Comment by RDKRACZ** [comment]
@kb-1000 Tried it on Arch... I really don't know what's happening but still doesn't work.
![Screenshot from 2021-08-28 22-24-52](https://user-images.githubusercontent.com/43277609/131230223-8161c46c-28e1-48ef-b768-f39baa1f2b78.png)
